src/commands/flow/internal/saveToCsv.ts

```diff
@@ -0,0 +1,63 @@
+import { createFile, fileExists, getRealPath, readFile } from '@sasjs/utils'
```

Comment:
would be nice to move it to `@sasjs/utils`

Reply:
Raised ticket for it: https://github.com/sasjs/cli/issues/921